go/tasks/plugins/svc/service_client.go

```diff
@@ -0,0 +1,15 @@
+package svc
```

Comment:
If we are not doing service plugin implementation for now, let's move this inside Presto... I think this interface needs some rework to make it generic....  @wild-endeavor what do you think?

Reply:
I think @wild-endeavor suggested this yesterday. I'm fine moving it.